metron-streaming/Metron-Common/src/test/java/org/apache/metron/enrichment/EnrichmentConfigTest.java

```diff
@@ -0,0 +1,253 @@
+package org.apache.metron.enrichment;
+
+import org.apache.metron.Constants;
+import org.apache.metron.domain.SourceConfig;
+import org.apache.metron.utils.JSONUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class EnrichmentConfigTest {
+  @Test
+  public void testThreatIntel() throws Exception {
+    /*
+    {
+      "index": "bro",
```

Comment:
I'd get rid of them too. I suspect they'll quickly get out of sync and become misleading.  Maybe go ahead and add in multiline or just use a resource file?